kf_lib_data_ingest/etl/load/load.py

```diff
@@ -456,19 +458,34 @@ def _run(self, target_entities):
 
             self.prev_entity = None
             total = len(entities)
+
+            if self.use_async:
+                ex = concurrent.futures.ThreadPoolExecutor()
+
             for i, entity in enumerate(entities):
                 entity_id = entity['id']
                 endpoint = entity['endpoint']
                 body = entity['properties']
                 links = entity['links']
 
+                verb = 'Queueing' if self.use_async else 'Loading'
                 self.logger.info(
-                    f'Loading {entity_type} {entity_id} (#{i+1} of {total})'
-                )
-                self._load_entity(
-                    entity_type, endpoint, entity_id, body, links
+                    f'{verb} {entity_type} {entity_id} (#{i+1} of {total})'
```

Comment:
Hmm now that we're loading async, seeing the current count of total in this log message is not as useful bc the Queueing log messages appear in batches in the log. 

Could we modify the log message on https://github.com/kids-first/kf-lib-data-ingest/blob/d0071ea0049c089bcc9d53c8f6203adc668c6074/kf_lib_data_ingest/etl/load/load.py#L389 so that it reports the current load count of the total?

So this:
```
Loaded participant 30 --> {kf_id: PT_EGE893F9}
```

would become: 
```
Loaded participant 30 --> {kf_id: PT_EGE893F9} (#605 of 2310)
```

When the log messages are scrolling by super fast, you're usually seeing those ^ log messages so seeing the load count out of the total gives you a good idea of progress.

Reply:
√

Reply:
https://github.com/kids-first/kf-lib-data-ingest/pull/344/commits/7538d86af1466ac180b49ccf5503d3448ea22c22